### PR TITLE
Init bundling element in the mock response.

### PR DESCRIPTION
**Pull request by shinfan**
Fixes: https://github.com/googleapis/toolkit/issues/417

### CODE REVIEW

File: src/main/java/com/google/api/codegen/transformer/InitCodeTransformer.java [hunk 1]
```diff
@@ -162,9 +162,15 @@ private GapicSurfaceTestAssertView createAssertView(String expected, String actu
...
     if (context.getMethodConfig().isPageStreaming()) {
+      // Initialize one resource element if it is page-streaming.
       PageStreamingConfig config = context.getMethodConfig().getPageStreaming();
       fields.add(config.getResourcesField().getSimpleName() + "[0]");
     }
+    if (context.getMethodConfig().isBundling()) {
+      // Initialize one bundling element if it is bundling.
+      fields.add(
+          context.getMethodConfig().getBundling().getSubresponseField().getSimpleName() + "[0]");
+    }
```
Comment (garrettjonesgoogle) [inline src/main/java/com/google/api/codegen/transformer/InitCodeTransformer.java]: Why don't we just initialize the whole object tree, and including at least one element of every list?

Comment (shinfan) [inline src/main/java/com/google/api/codegen/transformer/InitCodeTransformer.java]: Yes the alternative would be init the all the fields but I feel the marginal benefits of initializing more fields is small. Personally I would prefer the generated code as short as possible given the similar functionaliry. Thoughts?

Comment (garrettjonesgoogle) [inline src/main/java/com/google/api/codegen/transformer/InitCodeTransformer.java]: Hmm I think you make a good point. Let's go with this.

**Comment by garrettjonesgoogle** [comment]
LGTM

**Pull request merged**